unstoppable-ios-app/domains-manager-ios/Extensions/Task.swift

```diff
@@ -8,8 +8,8 @@
 import Foundation
 
 extension Task where Success == Never, Failure == Never {
-    static func sleep(seconds duration: TimeInterval) async throws {
+    static func sleep(seconds duration: TimeInterval) async {
         let duration = UInt64(duration * 1_000_000_000)
-        try await Task.sleep(nanoseconds: duration)
+        try? await Task.sleep(nanoseconds: duration)
```

Comment:
which `duration` you want to use? The passed one or local one?

Reply:
Updated variable name to make it more clear